apps/src/templates/CensusForm.jsx

```diff
@@ -1,9 +1,13 @@
 import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
 import Checkbox from './forms/Checkbox';
 import {UnconnectedCensusFollowUp as CensusFollowUp} from './CensusFollowUp';
 import Button from './Button';
 import color from "../util/color";
 import i18n from "@cdo/locale";
+import '../sites/studio/pages/schoolInfo';
```

Comment:
JS files in /pages are meant to be entry points that should not be imported from elsewhere. If we need this code in multiple places, we're probably better off sticking it somewhere else and then importing both into pages/schoolInfo and into here

It would also be nice to have a comment next to that code about what its purpose is, as it's looks to be doing a bunch of jquery stuff that is tightly coupled to the DOM structure.